Sources/Web3Core/Structure/Event+Protocol.swift

```diff
@@ -77,7 +77,7 @@ public enum Networks {
 
     static let allValues = [Mainnet, Ropsten, Kovan, Rinkeby]
 
-    public static func fromInt(_ networkID: UInt) -> Networks? {
+    public static func fromInt(_ networkID: UInt) -> Networks {
```

Comment:
We never really returned `nil` from here or at least we do not return any `nil`s now.